util/__init__.py

```diff
@@ -140,34 +148,47 @@ def __exit__(self, *args, **kwargs):
 
 @contextlib.contextmanager
 def show_mail_on_error():
-    if six.PY3:
-        import faulthandler
-
-        faulthandler.enable()
-        with contextlib.suppress(AttributeError, ImportError):
-            import signal
-
-            faulthandler.register(signal.SIGUSR2, all_threads=True)
+    faulthandler.enable()
+    with contextlib.suppress(AttributeError, ImportError):
```

Comment:
```suggestion
    with contextlib.suppress(AttributeError):
```
`signal` should be available on all platforms - also you've floated it anyway, so this does nothing